### PR TITLE
Remove code associated with openCta

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

This started somewhere completly different: I wanted to convert `lib/clamp` to ES6. Turned out, that all the code which could call that module was behind a (very) old switch, which seems not to be used anymore. @johnduffell mentioned this was an experiment once, but was turned off soon. No one else knows so far what `openCta` could be.

I doubt that 

`clamp(this.getElem('body'), 10, true);`

should ever be executed, since it would reduce the height of the body element to 10 lines. 😱 

## What is the value of this and can you measure success?

Less code.

## Does this affect other platforms - Amp, Apps, etc?

Discussion.

## Screenshots

No.

## Tested in CODE?

No.

### CODE REVIEW

File: common/app/common/configuration.scala [hunk 1]
```diff
@@ -440,10 +440,6 @@ class GuardianConfiguration extends Logging {
     lazy val expiredPaidContentUrl = s"${site.host}/info/2015/feb/06/paid-content-removal-policy"
   }
 
-  object open {
```
Comment (NataliaLKB) [inline common/app/common/configuration.scala]: Wow! This is 4 years old https://github.com/guardian/frontend/pull/2382

File: common/app/common/configuration.scala [hunk 2]
```diff
@@ -440,10 +440,6 @@ class GuardianConfiguration extends Logging {
...
-    lazy val ctaApiRoot = configuration.getMandatoryStringProperty("open.cta.apiRoot")
```
Comment (TBonnin) [inline common/app/common/configuration.scala]: Don't forget to remove the `open.cta.apiRoot` entry in the config file?

**Comment by johnduffell** [comment]
after 4 years only github remembers where it came from :1st_place_medal:

**Review by NataliaLKB — APPROVED** [review]
Wow! Good find. I am not aware of any reason why we want to keep this 4 year old test when the switch is off. 

Maybe double check with @stephanfowler though.

**Review by TBonnin — APPROVED** [review]
I am happy if the compiler is happy :)
Thank you for doing this @gustavpursche

File: common/app/common/configuration.scala (hunk 2 above)
Comment (gustavpursche) [inline common/app/common/configuration.scala]: @TBonnin Good catch. Deleted in https://github.com/guardian/frontend/pull/17404/commits/da23db2ec8d1a213f10b097dfdc1ee237c33a30e

Comment (TBonnin) [inline common/app/common/configuration.scala]: The config files lives on S3. Here are the manual step to update it https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/12-Update-configuration-in-s3.md

The DEVINFRA file is only used when running the app on Teamcity. It set to empty or fake values some mandatory config parameters.

Comment (gustavpursche) [inline common/app/common/configuration.scala]: Ah, ooh. Thanks - didn't know about this. 👍

File: common/app/common/configuration.scala [hunk 3]
```diff
@@ -65,7 +65,7 @@ object GuardianConfiguration extends Logging {
   lazy val configuration = {
     // This is version number of the config file we read from s3,
     // increment this if you publish a new version of config
-    val s3ConfigVersion = 34
+    val s3ConfigVersion = 35
```
Comment (TBonnin) [inline common/app/common/configuration.scala]: 👍

File: common/app/common/configuration.scala (hunk 2 above)
Comment (gustavpursche) [inline common/app/common/configuration.scala]: Update S3 configuration on https://github.com/guardian/frontend/pull/17404/commits/e02f8c407c4de3f9b9178d01a2f283bc1f6ad8de ✨ (great docs!)

**Comment by stephanfowler** [comment]
👍  I have zero recollection of what openCta was!

**Pull request merged**